src/web/components/ClientSideTokenGeneration/CstgDomainsTable.scss

```diff
@@ -25,10 +25,9 @@
       display: flex;
       justify-content: flex-end;
       flex-wrap: wrap;
-    }
 
-    .action-cell > button {
-      padding: 5px 0 5px 10px;
+      
+   
```

Comment:
there's a few blank lines we can delete here

Reply:
Sorry about that I assumed prettier would fix it but I think it was broken on my machine yesterday